packages/@react-facet/core/src/hooks/useFacetCallback.ts

```diff
@@ -66,14 +62,16 @@ export function useFacetCallback<M, Y extends Facet<unknown>[], T extends [...Y]
   // eslint-disable-next-line react-hooks/exhaustive-deps
   return useCallback(
     (...args: K) => {
-      const values = facetsRef.current
+      const values = facets.map((facet) => facet.get())
```

Comment:
Should this be args.map()?

I'm not sure how to fix errors the the linter is throwing here.

Reply:
We should do it in the same way as in the `useLayoutEffect` above. Just pushed a new commit.